packages/assistify-help-request/server/hooks/searchHooks.js

```diff
@@ -0,0 +1,21 @@
+/* global SystemLogger */
+function onSpotlighStartHandler(item) {
+	SystemLogger.debug('onSpotlightStart', item.text);
+}
+
+function onSpotlighEndHandler(item) {
+	SystemLogger.debug('onSpotlightEnd', item.text);
+	if (item.result && item.result.rooms) {
+		const subscripedChannels = RocketChat.models.Subscriptions.findByUserId(Meteor.userId()).fetch();
+		const publicChannelsNotContaingUser = RocketChat.models.Rooms.findAllPublicNotContainingUserName(Meteor.user().username).fetch();
+		const accessibleRooms = subscripedChannels.concat(publicChannelsNotContaingUser);
+		const conversations = Meteor.call('searchConversations', item.text, accessibleRooms);
```

Comment:
@lobo87 the `accessibleRooms` should be determined and propagated by the callback itself, not within the implementation.

Reply:
see my answer two comment below